src/app/api/orders/route.ts

```
@@ -1,168 +1,168 @@
+            price: product.price,
+            image: product.image,
+            originalId: product.originalId
+          } : null
+        };
+      })
+    );
+    
+    return NextResponse.json({
+      ...order.toObject(),
+      items: populatedItems
+    });
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)